src/components/measureLanding/measureList/MeasureList.tsx

```diff
@@ -312,7 +312,11 @@ export default function MeasureList(props: {
             variant="outline-secondary"
             name="Select"
             onClick={(e) => {
```

Comment:
are you asking why we are sending the featureFlags?.qdmExport as prop to the onClick function this is because tanstack table is not being updating whenever the value of featureFlags is changes so it is always setting it to false so I had to send prop to fix this issue or else other solution would be making it this feature flag into a state variable and then using it

Reply:
no, I was just reducing it to single line since we are using arrow function

Reply:
Got it, I just pushed the code with the changes